Android/app/src/main/res/navigation/nav_main_app.xml

```diff
@@ -13,13 +13,25 @@
 
         <action
             android:id="@+id/action_FirstFragment_to_SecondFragment"
```

Comment:
are we going to rename this to `action_HomePostFragment_to_eventsFragment`?

Reply:
Yeah, sorry about that!